Makefile.am

```diff
@@ -972,9 +974,11 @@ imagetopdf_SOURCES = \
 	filter/imagetopdf.c
 imagetopdf_CFLAGS = \
 	$(CUPS_CFLAGS) \
+	$(EXIF_CFLAGS) \
 	-I$(srcdir)/cupsfilters/
 imagetopdf_LDADD = \
 	$(CUPS_LIBS) \
+	$(EXIF_LIBS) \
 	libcupsfilters.la
 
 imagetops_SOURCES = \
```

Comment:
You do not need this here for the individual filter executables, it is only needed in libcupsfilters itself, as you correctly did.  So remove these additions here.